src/update_lmod_system_cache_files.in

```diff
@@ -334,7 +334,7 @@ set_defaults
 check_parameters
 
 if [ -z "$LMOD_REVERSEMAPT_DIR" ]; then
-  LMOD_REVERSEMAPT_DIR="$LMOD_CACHE_DIR/../reverseMapD"
+  LMOD_REVERSEMAPT_DIR="$LMOD_CACHE_DIR/reverseMapD"
```

Comment:
Because now you can run: `update_lmod_system_cache_files` and it does everything nice, expect it also escapes the cache directory. If you want everything put below one dir, you need to specify the cache dir on the commandline while now it fetches it from the config.